mautrix/util/proxy.py

```diff
@@ -90,17 +90,19 @@ async def proxy_with_retry(
     proxy_handler: ProxyHandler,
     on_proxy_change: Callable[[], Awaitable[None]],
     max_retries: int = 10,
+    min_wait_seconds: int = 60,
+    retryable_exceptions: Iterable[Exception] = RETRYABLE_PROXY_EXCEPTIONS,
```

Comment:
probably better to just define it as a tuple

```suggestion
    retryable_exceptions: tuple[Exception] = RETRYABLE_PROXY_EXCEPTIONS,
```